packages/lit-dev-content/src/components/litdev-playground-page.ts

```diff
@@ -104,13 +161,20 @@ export class LitDevPlaygroundPage extends LitElement {
     };
     this._playgroundProject = mustFindChild('playground-project')!;
     this._playgroundPreview = mustFindChild('playground-preview')!;
+    this._fileEditor = mustFindChild('playground-file-editor')!;
     this._shareButton = mustFindChild('litdev-playground-share-button')!;
     this._downloadButton = mustFindChild('litdev-playground-download-button')!;
     this._examplesDrawer = mustFindChild('litdev-drawer')!;
     this._exampleControls = mustFindChild('litdev-example-controls')!;
     this._examplesDrawerScroller = mustFindChild(
       'litdev-drawer .minimalScroller'
     )!;
+    // Unforuntately, the file editor does not emit a change event, so we reach
```

Comment:
Oh, hmm. I think that event should be `composed` maybe? Or we should propagate it from `file-editor` (and also from `ide`). This is fine for now though.

Reply:
firing a "change" event that bubbles and composed sounds like trouble